code/game/objects/items/weaponry.dm

```diff
@@ -294,7 +294,8 @@ for further reading, please see: https://github.com/tgstation/tgstation/pull/301
 		if (user.getStaminaLoss() >= 100)
 			user.throw_at(landing_turf, 2, 2)
 			user.Paralyze(4 SECONDS)
-			user.visible_message(span_warning("You're too tired tired to finish the roll!"))
+			to_chat(user, span_warning("You're too tired tired to finish the roll!"))
+			user.visible_message(span_notice("[user] collapses on the ground, exhausted!"))
```

Comment:
could use just one line here
user.visible_message(span_notice("[user] collapses on the ground, exhausted!"), span_warning("You're too tired to finish the roll!"))

also, typo of "tired tired"